addons/website/static/src/scss/website.scss

```diff
@@ -450,10 +457,15 @@ table.table_desc tr td {
 
 // Three columns
 
-.s_three_columns {
+.s_three_columns, .s_comparisons {
     .align-items-stretch .card {
         height: 100%;
     }
+
+    .card {
+        // Allows to set border radius without overflow.
+        overflow: hidden;
+    }
```

Comment:
Probably a bad idea, if an user adds a dropdown in there it will not be able to overflow (already the case with parallax and I'd like to find another way in master there too)

Reply:
An alternative is to set the corresponding border-top radius on the top image and border-bottom radius on the card body, seems like this would benefit from the feature "selecting different classes on different sub elements with one option" (except it would need to work not only with selectClass, but I have some ideas for that).

Would you like me to implement a workaround in JS in the meantime?

Reply:
I am not sure there is non crappy solution for this... maybe the correct solution is to verify the spec, do we really want to allow overriding BS card radius and border width :/